tests/roots/test-base/target/example_generics.py

```diff
@@ -24,7 +24,7 @@ class Response(GenericModel, Generic[DataT]):
     data: Optional[DataT]
     error: Optional[Error]
 
-    @validator('error', always=True)
+    @field_validator('error', always=True)
```

Comment:
By the migration docs, it seems `field_validator` doesn't have `always` anymore, but I couldn't find what would be equivalent